x/id/internal/types/codec.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // RegisterCodec registers concrete types on wire codec
 func RegisterCodec(cdc *codec.Codec) {
-	cdc.RegisterConcrete(MsgSetIdentity{}, fmt.Sprintf("%s/SetIdentity", ModuleName), nil)
+	cdc.RegisterConcrete(MsgSetIdentity{}, fmt.Sprintf("%s/SetIdentity", "commercio"), nil)
```

Comment:
I think we can avoid using the `fmt.Sprintf` method at all, replacing it with a simple "commercio/SetIdentity` string